examples/provider/main.tf

```diff
@@ -1,7 +1,8 @@
 terraform {
   required_providers {
     rhoas = {
-      source  = "pmuir/rhoas"
+      source  = "registry.terraform.io/redhat-developer/rhoas"
+      version = "0.1"
```

Comment:
We are currently on `0.3.0`

Reply:
Maybe you can change it in the `docs/index.md` as that is out of date aswell

Reply:
Sure, updated all usages of `"0.1"` to `"0.3"` 👍 
It should be nice with another PR to automate this replace as part of the release process :)